mandible/metadata_mapper/exception.py

```diff
@@ -1,3 +1,6 @@
+RAISE_EXCEPTION = object()
```

Comment:
This should be defined in `key.py` as it exists exclusively to be the default value for the `default` argument of `Key`.